src/main/cliapp/src/containers/experimentalConditionsPage/ExperimentalConditionsTable.js

```diff
@@ -119,18 +119,25 @@ export const ExperimentalConditionsTable = () => {
         delete event.data.conditionGeneOntology;
       }
       let updatedRow = JSON.parse(JSON.stringify(event.data));//deep copy
-      if (event.data.conditionGeneOntology && Object.keys(event.data.conditionGeneOntology).length >= 1) {
-          event.data.conditionGeneOntology.curie = trimWhitespace(event.data.conditionGeneOntology.curie);
-          updatedRow.conditionGeneOntology = {};
-          updatedRow.conditionGeneOntology = event.data.conditionGeneOntology;
+      
+      const curieFields = ["conditionClass", "conditionId", "conditionAnatomy", "conditionTaxon", "conditionGeneOntolgy", "conditionChemical"];
+      for (var ix = 0; ix < curieFields.length; ix++) {
+        if (event.data[curieFields[ix]] && Object.keys(event.data[curieFields[ix]]).length >= 1) {
+          event.data[curieFields[ix]].curie = trimWhitespace(event.data[curieFields[ix]].curie);
+          updatedRow[curieFields[ix]] = {};
+          updatedRow[curieFields[ix]] = event.data[curieFields[ix]];
+        }
       }
-
       mutation.mutate(updatedRow, {
           onSuccess: (data, variables, context) => {
```

Comment:
Might be better here to change `data` to `response` that was later on `data.data` turns into `response.data`

Reply:
@oblodgett - done, and OntologyTerm endpoints added